src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The `href` attribute currently links to `styles/main.scss`. This should be updated to link to the compiled CSS file `css/main.css` to ensure the styles are applied correctly in the browser.